src/home_robot/home_robot/agent/ovmm_agent/ovmm_agent.py

```diff
@@ -65,7 +65,7 @@ def __init__(self, config, device_id: int = 0):
         self.skip_skills = config.AGENT.skip_skills
         self.max_pick_attempts = 10
         if config.GROUND_TRUTH_SEMANTICS == 0:
-            self.semantic_sensor = OvmmPerception(config, device_id)
+            self.semantic_sensor = OvmmPerception(config, device_id, self.verbose)
```

Comment:
This should work, but that's odd that the verbocity of the agent is controlled by `config.AGENT.PLANNER.verbose`. 

From ObjectNavAgent
```
self.verbose = config.AGENT.PLANNER.verbose
```

Reply:
Yes, we need to pull it out. I can do it here - this will require changes in all sim/real configs.

Reply:
Ok, let's handle this later.